### ISSUE TITLE
Document application webroot setting

### ISSUE THREAD

**Issue by ctgraham** [ulsdevteam/ccvgd-frontend#7]
The application expects to be deployed in the webroot, and will reference the javascript includes via an absolute path.

For example, deploying to "http://localhost/ccvgd/" will look for `runtime-es2015.a1ac991fc00ebf39524f.js` in "http://localhost/runtime-es2015.a1ac991fc00ebf39524f.js"

This should be configurable, or should be documented within the readme.

**Comment by Xiaoxin-He** [comment]
Hey Dr. Clinton!  for deploy on application base for router, do with command line 
````
ng build --prod --base-href /ccvgd/
````
on `dist/CCVGproject/index.html` line 6 will be updated with 

<img width="831" alt="Screen Shot 2021-07-26 at 5 36 22 PM" src="https://user-images.githubusercontent.com/43584974/127062465-38e93086-70f2-4733-9b85-fd687d44f9b2.png">

which the app could listening on 

````
http://localhost/ccvgd/
````
see more information on the readme, thanks!

**Comment by Xiaoxin-He** [comment]
> The application expects to be deployed in the webroot, and will reference the javascript includes via an absolute path.
> 
> For example, deploying to "http://localhost/ccvgd/" will look for `runtime-es2015.a1ac991fc00ebf39524f.js` in "http://localhost/runtime-es2015.a1ac991fc00ebf39524f.js"
> 
> This should be configurable, or should be documented within the readme.

**Comment by ctgraham** [comment]
Resolved by 5e62deb.
